assets/css/style.css

```diff
@@ -0,0 +1,251 @@
+:root {
```

Comment:
Just wanted to say how much I enjoyed the styling of the site. It was really good. Well done!

Reply:
Thanks Lee!!! Long live rock n roll!!!!